### PR TITLE
django automation

### PR DESCRIPTION
**Pull request by yaelerdr**
Added a script for the VM to run - setup.sh.
Changed VagarantFile so it will run the setup.sh,and changed port to 8000
Added Pipefile and Pipefile.lock

### CODE REVIEW

**Review by benmenachem — APPROVED** [review]

**Review by eldaraharonson — APPROVED** [review]
Looks good!

File: setup.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# Install Pipenv, the -n option makes sudo fail instead of asking for a
+# password if we don't have sufficient privileges to run it
+sudo -n dnf install -y pipenv
+
+cd /vagrant
+# Install dependencies with Pipenv
+pipenv sync --dev
+
+# Run database migrations
+pipenv run python manage.py migrate
```
Comment (guy9050) [inline setup.sh:14]: why is `manage.py` not included in the PR? if we merge it like that, it will cause failures

**Comment by guy9050** [comment]
Seems like some file are missing/ were not added to the PR (settings.py, mange.py.. etc)

File: setup.sh (hunk 1 above)
Comment (yaelerdr) [reply setup.sh:14]: Will do

**Pull request closed without merging**